### PR TITLE
RAIN-3096 3097 3115 3122 3099, TL Employee flow issue fixes 

### PR DESCRIPTION
**Pull request by saurabh-egov**

### CODE REVIEW

File: packages/modules/tl/src/components/inbox/MobileInbox.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+}) => {
+  const { t } = useTranslation();
+  const getData = () => data?.table.map(e => ({
+      [t("WF_INBOX_HEADER_APPLICATION_NO")]:e?.["applicationId"],
+      [t("TL_COMMON_TABLE_COL_APP_DATE")]:convertEpochToDateDMY(e?.["date"]),
+      [t("WF_INBOX_HEADER_LOCALITY")]:t(Digit.Utils.locale.getRevenueLocalityCode(e?.["locality"], e?.["tenantId"])),
+      [t("WF_INBOX_HEADER_STATUS")]:t(e?.["status"]),
+      [t("WF_INBOX_HEADER_CURRENT_OWNER")]:t(e?.owner),
+      [t("WF_INBOX_HEADER_SLA_DAYS_REMAINING")]:e?.["sla"]}))
+  debugger
```
Comment (jagankumar-egov) [inline packages/modules/tl/src/components/inbox/MobileInbox.js:37]: can you remove debuggers

File: packages/modules/tl/src/components/SearchApplication.js [hunk 2]
```diff
@@ -89,19 +89,7 @@ const SearchApplication = ({tenantId, t, onSubmit, data }) => {
...
-                            <span className="link">
-                                <Link to={`/digit-ui/employee/tl/application-details/${row.original["applicationNumber"]}`}>
-                                    {row.original["licenseNumber"]}
-                                </Link>
-                            </span>
-                            : "-"}
-                    </div>
-                );
-            },
+            accessor: (row) => GetCell(row.licenseNumber || "-"),
```
Comment (jagankumar-egov) [inline packages/modules/tl/src/components/SearchApplication.js:92]: is navigation to application details is handled?

File: packages/modules/tl/src/components/inbox/search.js [hunk 3]
```diff
@@ -28,6 +28,7 @@ const SearchApplication = ({ onSearch, type, onClose, searchFields, searchParams
   useEffect(() => {
     searchFields.forEach(({ pattern, name, maxLength, minLength, errorMessages, ...el }) => {
       const value = form[name];
+      debugger
```
Comment (jagankumar-egov) [inline packages/modules/tl/src/components/inbox/search.js:31]: remove debugger

File: packages/modules/tl/src/components/SearchApplication.js (hunk 2 above)
Comment (saurabh-egov) [reply packages/modules/tl/src/components/SearchApplication.js:92]: no cant navigate to license number as application details page takes application number

**Review by jagankumar-egov — APPROVED** [review]

**Pull request merged**